spec.md

```diff
@@ -1398,28 +1391,94 @@ module MyModule :
   connect myoutput, myinput
 ```
 
-In order for a connection to be legal the following conditions must hold:
+In the syntax for a `connect` statement, the first argument is referred to as the "right-hand side".
```

Comment:
well, that's all very confusing. Note that `connect rhs, lhs` used to be written as `lhs <- rhs` where that made the most sense. Maybe a footnote for this fact to help readers understand why that is the case (or we rewrite all this??)

Reply:
I might just change it to a more meaningful name. The "target" and the ... something? I'll see if anyone has any ideas.

Reply:
i realized after i wrote this that you had written some text below this that can help. I still think a LHS and RHS is ok with the rationale